shacl12-overview/index.html

```diff
@@ -449,6 +451,9 @@ <h2>SHACL-SHACL</h2>
 				<dt><a href="">SHACL-SHACL Profiling</a></dt>
 				<dd>Profiling specification elements only</dd>
 			</dl>
+			<p class="note">
+				There is no SHACL-SHACL graph for SHACL Compact Syntax since the specification defines an RDF syntax specialised for describing SHACL concepts - and does not introduce any new concepts itself.
```

Comment:
```suggestion
				There is no SHACL-SHACL graph for SHACL Compact Syntax since the specification defines an RDF syntax specialised for describing SHACL concepts, and does not introduce any new concepts itself.
```